bridgebot/bidding.py

```diff
@@ -24,23 +126,50 @@ def __init__(self, first_bidder):
             Players.WEST: []
         }
 
-    def __ns_first_bid(self, strain):
-        return
-        self.ns_first_bid = {
-            Strains.CLUBS: None,
-            Strains.DIAMONDS: None,
-            Strains.HEARTS: None,
-            Strains.SPADES: None,
-            Strains.NT: None
+        # Not necessary but makes it more human readable
```

Comment:
Just hard coding this to eliminate the confusion over where to put these dashes.